ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -1345,22 +1377,6 @@ func (h *FaultHandler) stopTCFault(ctx context.Context, taskMetadata *state.Task
 		field.CommandString: tcDeleteQdiscParentCommandComposed,
 		field.CommandOutput: string(cmdOutput[:]),
 	})
-	tcDeleteFilterCommandComposed := nsenterPrefix + fmt.Sprintf(tcDeleteFilterCommandString, interfaceName)
-	cmdList = strings.Split(tcDeleteFilterCommandComposed, " ")
-	cmdOutput, err = h.runExecCommand(ctx, cmdList)
-	if err != nil {
-		logger.Error("Command execution failed", logger.Fields{
-			field.CommandString: tcDeleteFilterCommandComposed,
-			field.Error:         err,
-			field.CommandOutput: string(cmdOutput[:]),
-			field.TaskARN:       taskMetadata.TaskARN,
-		})
-		return err
-	}
-	logger.Info("Command execution completed", logger.Fields{
-		field.CommandString: tcDeleteFilterCommandComposed,
-		field.CommandOutput: string(cmdOutput[:]),
-	})
```

Comment:
What was this for and why is it no longer needed?

Reply:
Originally we run 3 commands to delete a latency/packet loss fault:
1. tc qdisc del dev <interfaceName> parent 1:1 handle 10:
2. tc filter del dev <interfaceName> prio 1
3. tc qdisc del dev <interfaceName> root handle 1: prio
Now that we're using 2 different prio's (1 for allowlist, 2 for ips to be added to the filter), in step 2, we would need to do either of the following:
1. Delete both prio1 and prio2
2. Don't delete any of them. Everything will be cleaned up by the third command (when the root is deleted).
Both options work the same. After discussing this with Xing, we decided to go with option 2. When we delete prio that doesn't exist, there will be extra effort needed to handle the Linux error. When the customer doesn't add a filter, this will be triggered. Thus, we will simply rely on deleting the root handle to remove fault

Reply:
Thanks for the explanation. Sounds good!